modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -1419,7 +1419,7 @@ label:enterprise-edition[Enterprise Edition]
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|Enables shuffling of the returned load balancing result.
+a|Vary the order of the entries in routing tables each time one is produced. This means that different clients should select a range of servers as their first contact, reducing the chance of all clients contacting the same server if alternatives are available.
```

Comment:
@nick-giles-neo, this change must be reflected in the codebase as well in order to be merged.